docs/javascript/v2/features/error-handling.mdx

```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
+| **4001**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4002**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4003**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4004**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4005**       | ICE Connection Failed due to network issue                                                                | Mention user to check their network connection or try again after some time.                                |
+| **5000**       | Server error for Join                                                                                     | TODO                                                                                                        |
+| **5001**       | Trying to join a room which is already joined                                                             | Trying to join an already joined room.                                                                      |
+| **5002**       | Trying to join a room when preview is in progress                                                         | Call join action only when room state is "Preview" and not "Connecting".                                    |
+| **6000**       | Calling actions when not connected/joined to a room                                                       | Call the required action after joining a room.                                                              |
+| **6001**       | Unknown signalling error: {action} {error_info}                                                           | TODO                                                                                                        |
+| **6002**       | WebRTC Error: Error while renegotiating                                                                   | TODO                                                                                                        |
+| **6003**       | WebRTC engine is not ready yet                                                                            | TODO                                                                                                        |
```

Comment:
Wait for preview to complete before calling join. 


we should probably update the message. webRTC engine not ready seems vague